app/src/main/res/menu/downloads_completed.xml

```diff
@@ -16,6 +16,12 @@
         android:title="@string/downloads_log_label"
         custom:showAsAction="always" />
 
+    <item
+        android:id="@+id/action_delete_downloads_played"
```

Comment:
I would only show the menu item on the downloads screen. I think it now shows on all screens (untested)